app/views/tools/_tool.html.erb

```diff
@@ -1,8 +1,87 @@
+<script src="https://code.jquery.com/jquery-3.6.0.min.js"></script>
+<style>
+  .popup {
+    position: fixed;
+    top: 0;
+    left: 0;
+    height: 100%;
+    width: 100%;
+    background-color: rgba(0, 0, 0, 0.5);
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    z-index: 999;
+  }
+
+  .popup img {
+    max-height: 90%;
+    max-width: 90%;
+  }
+  .popup .close {
+    position: absolute;
+    bottom: 10px;
+    right: 10px;
+  }
+
+  body > *:not(.popup) {
+    z-index: 1;
+  }
+
+  .tool-image {
+  cursor: pointer;
+  }
+</style>
+
 <tr <%= raw 'class="warning"' if tool.is_checked_out? %>>
   <td><%= link_to tool.barcode, tool, class: 'btn btn-info btn-xs' %></td>
   <td><%= tool.name %></td>
   <td><%= tool.description %></td>
+  <td>
+    <%# add the image of the tool in its own column %>
+    <%= image_tag("tool_images/#{tool.name}.png", id: "tool-image-#{tool.name}", class: 'tool-image', data: { tool_name: tool.name, image_url: asset_path("tool_images/#{tool.name}.png") }, size: '70x70') %>
+ </td>
   <td><%= date_and_time(tool.checkouts.current.first.checked_out_at) if tool.is_checked_out? %></td>
   <td class="hidden-xs <%= 'hidden' if page == 'org' %>"><%= link_to tool.checkouts.current.first.organization.short_name, tool.checkouts.current.first.organization if tool.is_checked_out? %></td>
   <td class="visible-lg <%= 'visible-sm visible-md' if page == 'org' %>"><%= link_to tool.checkouts.current.first.participant.name, tool.checkouts.current.first.participant if tool.is_checked_out? && tool.checkouts.current.first.participant.present? %></td>
 </tr>
+
+
+<script>
+  // opens the image in a popup when it's clicked 
+  // closes the image when the "Close Image" button is clicked
+  function openPopup(imageUrl) {
+  if (document.querySelector('.popup')) {
+    // popup is already open, do nothing
+    return;
+  }
+  console.log("Popup opened with URL:", imageUrl);
```

Comment:
nit: `console.log`

Reply:
Ahh yeah that is a good point. As of right now, I think that as long as there is no tool without an image, there shouldn't be any errors, but that is definitely not the most ideal if changes are going to be made to the tools inventory. I'll try to find a way so that the page doesn't error if there isn't an image (also open for suggestions on how to do that if you have any :))

Reply:
I think you could use a `picture` html tag with an additional source, but not sure it's required, I didn't pull the repo and run it locally, but there's a chance it's fine as-is and will just be blank if there's no image